### ISSUE TITLE
Do you have Caffe implementation?

### ISSUE THREAD

**Issue by chenbinghui1** [syfafterzy/PCB_RPP#5]
I implement the IDE model with caffe, but I just achieve 79.6 R@1 and 56.4 mAP on Market1501 (SQ).

**Comment by syfafterzy** [comment]
Your results about IDE on Market-1501 are actually close to ours. With the bottlenet structure (256d fc+ BN), we achieve about 81 rank-1 accuracy and 62 mAP. We are not sure about the reason yet.

**Comment by chenbinghui1** [comment]
> Your results about IDE on Market-1501 are actually close to ours. With the bottlenet structure (256d fc+ BN), we achieve about 81 rank-1 accuracy and 62 mAP. We are not sure about the reason yet.

emm, I also use the same bottolenet structure (drop(0.5)+256d fc + BN +relu+softmax), I think the difference might exists in the BN layer, BN implementations in Pytorch and Caffe seem different.

**Comment by hoangtuanvu** [comment]
@syfafterzy If I can do PCB-RPP in Tensorflow, I also get same result, right?
